lib/Manager.php

```diff
@@ -317,6 +317,30 @@ public function getRoomsLongerActiveSince(\DateTime $maxActiveSince): array {
 		return $rooms;
 	}
 
+	/**
+	 * @return Room[]
```

Comment:
```suggestion
	 * @return list<Room>
```

:see_no_evil: I'm damaged from https://github.com/nextcloud/spreed/pull/13830